frontend/apps/marketing/README.md

```diff
@@ -0,0 +1,11 @@
+# Code.org Marketing Application
+
+## Getting Started
+
+First, run the development server:
```

Comment:
Mention `yarn install`?

Reply:
My terminal is getting confused about node versions, and as a result when I switch to the newer one via `nvm`, I'm having to install yarn (since yarn is installed yarn versions are per node version ([I think?](https://stackoverflow.com/questions/52174381/yarn-not-installing-in-nvm-version-node-version)).

Anyway, I'm guessing we don't want node/yarn setup docs in every application's README, but could we point to a top-level `frontend` README that describes these requirements?

Reply:
Yeah I suspect we'll need to use the same node versions. I plan to upgrade Code.org to node 22 to match them up (since node 18 and 20 are EOL). Let me see if we can make it easier to transit between directories.